packages/transaction-controller/src/TransactionController.ts

```diff
@@ -622,6 +833,7 @@ export class TransactionController extends BaseControllerV1<
       sendFlowHistory,
       swaps = {},
       type,
+      networkClientId,
```

Comment:
Do we want this to be optional? If we're investing in this refactor, should we also update the clients as needed to always provide a network client ID?